runtime/common/src/constants.rs

```diff
@@ -24,3 +26,9 @@ pub const VARA_DECIMAL: u8 = 12;
 
 /// Vara Token Symbol
 pub const VARA_TOKEN_SYMBOL: &str = "VARA";
+
+/// The minimal amount of blocks to resume represented as a factor of weeks.
+pub const RENT_RESUME_WEEK_FACTOR: BlockNumber = 4;
```

Comment:
it's not block number type. maybe usize?

Reply:
with following cast on multiplication? 
or add bound for bn that can be multiplied by usize. up to you